steveDevonport/steve_devonport.md

```diff
@@ -0,0 +1,51 @@
+
+When is a hero not a hero?
+
+In a previous role I was tasked with creating the Test Automation for a number of different web services, however when I joined the team I quickly discovered that they were lacking the capability to deliver customer value rapidly, securely, or with predictability. All this was compounded by the Scrum ceremonies not being orchestrated and there was no Scrum Master. The technology and tooling side was no better. To present a balanced view my new team were delivering changes/new features into production...just very slowly.
+
+I could easily have walked away from this team and the role but I choose to stay mainly because I love a challenge, no matter how impossible it seems at first.
+
+As I got to know the people on the team, over the first several months, I put together a list of 'things' that were either missing or just seemed wrong. And so it was that before the end of my probation period I presented my manager with a three pronged approach that would allow us to deliver features transparently, at pace, and with reliability - all within 12 months.
```

Comment:
Sentence starts with an and, consider rewording.

Reply:
Good spot - fixed